content/v2/domains.markdown

```diff
@@ -52,6 +52,29 @@ List all domains for the account 1010 that have name matching "example":
           -H 'Accept: application/json' \
           https://api.dnsimple.com/v2/1010/domains?name_like=example
 
+### Sorting
+
+Sort records with the `sorting` param by specifing the field and the sorting policy: `asc` for ascending and `desc` for descending.
+Multiple fields are allowed and MUST be comma separated.
+
+Name | Description
+-----|------------
+`:id` | Sort domains by ID
```

Comment:
@jodosha the data in this table is different than the other places, for example params. In params I used `:account_id` because that is the name of the placeholder in the URI.

Here there is no placeholder at all, this is just a list of values.